config/locales/doc_auth/es.yml

```diff
@@ -199,6 +199,7 @@ es:
       document_capture: Incluir fotos de su identificación
       document_capture_back: Parte trasera de su documento de identidad
       document_capture_front: Parte delantera de su documento de identidad
+      document_capture_selfie:
```

Comment:
I'd expect the translations will be added before this is merged?

Reply:
💯 The translations will be added before merge.

Reply:
Translations are in ([comment](https://github.com/18F/identity-idp/pull/9580/commits/170080afa059600490eb617c1609bc7680682537)